qemu/tests/blockdev_commit_non_existed_base.py

```diff
@@ -0,0 +1,43 @@
+import logging
+
+from virttest.qemu_monitor import QMPCmdError
+
+from provider import backup_utils
+from provider.blockdev_commit_base import BlockDevCommitTest
+
+
+class BlockdevCommitNonExistedBase(BlockDevCommitTest):
+
+    def commit_snapshots(self):
+        device = self.params.get("device_tag")
+        device_params = self.params.object_params(device)
+        snapshot_tags = device_params["snapshot_tags"].split()
+        self.device_node = self.get_node_name(device)
+        options = ["base-node", "top-node", "speed"]
+        arguments = self.params.copy_from_keys(options)
+        arguments["base-node"] = 'sn0'
+        arguments["top-node"] = self.get_node_name(snapshot_tags[-2])
+        device = self.get_node_name(snapshot_tags[-1])
+        commit_cmd = backup_utils.block_commit_qmp_cmd
+        cmd, args = commit_cmd(device, **arguments)
+        try:
+            self.main_vm.monitor.cmd(cmd, args)
+        except QMPCmdError as e:
+            logging.info("Error message is %s" % e.data)
+            if self.params["qmp_error_msg"] not in str(e.data):
+                self.test.fail("Error message not as expected")
```

Comment:
Here we have to add 'else' , i.e. block-commit succeeded unexpectedly, then we should fail the case

Reply:
you may also need to override  run_test function here, for in base class, run_test also calls verify_data_file to do some md5sum check, which is not needed in your case

Reply:
hi Leidong, what's your opinion about this comment? this is a negative test, so we may not need the verify_data_file step

Reply:
Yes,will delete verify_data_file step.Thanks a lot.